src/hooks/appContext.js

```diff
@@ -0,0 +1,15 @@
+import React, { useContext } from 'react';
+
+export const AppContext = React.createContext();
+
+export const useSelector = selectorFn => {
+  const { state } = useContext(AppContext);
+  return selectorFn(state);
+};
+
+export const useDispatch = () => {
+  const { dispatch } = useContext(AppContext);
+  return dispatch;
+};
+
+export const getMarkdownState = state => state.markdown;
```

Comment:
call thing `getMarkdown` and put it in a selectors folder. Make sure to add a test